cfme/storage/volume_snapshot.py

```diff
@@ -0,0 +1,238 @@
+# -*- coding: utf-8 -*-
+import attr
+
+from navmazing import NavigateToSibling, NavigateToAttribute
+from widgetastic_manageiq import (
+    Accordion,
+    BaseEntitiesView,
+    BreadCrumb,
+    ItemsToolBarViewSelector,
+    ManageIQTree,
+    SummaryTable
+)
+from widgetastic_patternfly import Button, Dropdown, FlashMessages
+from widgetastic.widget import View, Text
+
+from cfme.base.ui import BaseLoggedInPage
+from cfme.common import TagPageView, WidgetasticTaggable
+from cfme.exceptions import SnapshotNotFoundError, ItemNotFound
+from cfme.modeling.base import BaseCollection, BaseEntity
+from cfme.utils.appliance.implementations.ui import CFMENavigateStep, navigator, navigate_to
+from cfme.utils.wait import wait_for
+
+
+class VolumeSnapshotToolbar(View):
+    """The toolbar on the Volume Snapshot page"""
+    policy = Dropdown('Policy')
+    download = Dropdown('Download')
+    view_selector = View.nested(ItemsToolBarViewSelector)
+
+
+class VolumeSnapshotDetailsToolbar(View):
+    """The toolbar on the Volume Snapshot detail page"""
+    configuration = Dropdown('Configuration')
+    policy = Dropdown('Policy')
+    download = Button('Download summary in PDF format')
+
+
+class VolumeSnapshotDetailsEntities(View):
+    """The entities on the Volume Snapshot detail page"""
+    breadcrumb = BreadCrumb()
+    properties = SummaryTable('Properties')
+    relationships = SummaryTable('Relationships')
+    smart_management = SummaryTable('Smart Management')
+
+
+class VolumeSnapshotDetailSidebar(View):
+    """The accordion on the Volume Snapshot details page"""
+    @View.nested
+    class properties(Accordion):  # noqa
+        tree = ManageIQTree()
+
+    @View.nested
+    class relationships(Accordion):  # noqa
+        tree = ManageIQTree()
+
+
+class VolumeSnapshotView(BaseLoggedInPage):
+    """A base view for all the Volume Snapshot pages"""
+    title = Text('.//div[@id="center_div" or @id="main-content"]//h1')
+    flash = FlashMessages(
```

Comment:
**Required**
This is provided by inherited `BaseLoggedInPage` view, remove duplicate widget here.

Reply:
right; flash widget removing. @mshriver its good to add title also in `BaseLoggedInPage` View as it's common for all pages; what you say?

Reply:
ah! glad you mentioned it.

Title is actually defined as part of the entities section of the page, in BaseEntitiesView.
https://github.com/ManageIQ/integration_tests/blob/master/widgetastic_manageiq/__init__.py#L2819